client/packages/host/src/api/hooks/help/useFeedbackForm.ts

```diff
@@ -38,13 +38,22 @@ export function useFeedbackForm() {
 const useInsert = () => {
   const { api, storeId } = useFeedbackFormGraphQL();
 
-  const mutationFn = async ({replyEmail, body}: ContactFormInput) => { 
+  const mutationFn = async ({contactFormType, replyEmail, body}: ContactFormInput) => { 
+    let contactType;
+    if (contactFormType == ContactFormNodeType.Feedback) {
+      contactType = ContactFormNodeType.Feedback;
+    }
+    if (contactFormType == ContactFormNodeType.Support) {
+      contactType = ContactFormNodeType.Support;
+    }
+    if (contactType == undefined) {throw Error("contactType is undefined")} // this is ok as the front end will disable the send button if this is the case.
```

Comment:
I think this `if` block is a bit redundant, and might be easy to miss if another option got added - nicer just to use the enum values throughout. Assuming this was done as a result of the `onChange` of the Select component handling a string rather than the enum value?

I think this is one of the cases where the `as` keyword is ok... put a suggestion and comment here, let me know if you have questions:

https://github.com/msupply-foundation/open-msupply/commit/20a3472c951500e3e36f2e0237cbfae111c6f179#r150471171

Also looks like there are a number of white space differences - we might need to take a look at your prettier configuration, I think it is not formatting your files on save 🙏

Reply:
Cool! I had a look at my formatter settings and I think they're correct now... we'll see in the future I suppose.
I'll deal with that if block now.